scripts/bootstrap

```diff
@@ -22,9 +22,9 @@ echo "using rust version '$rust_version'"
 # here we set the toolchain to 'none' and rustup will pick up on ./rust-toolchain.toml
 run curl --fail https://sh.rustup.rs -sSf | run sh -s -- -y --default-toolchain "none" --no-modify-path
 
-echo "looking for ic-cdk-optimizer"
-if ! command -v ic-cdk-optimizer
+echo "looking for ic-cdk-optimizer 0.3.4"
+if [[ ! "$(command -v ic-cdk-optimizer)" || "$(ic-cdk-optimizer --version)" == "ic-cdk-optimizer 0.3.4" ]]
```

Comment:
won't this try to install v0.3.4 if it's already installed, and do nothing if the version is different than 0.3.4?
```suggestion
if [[ ! "$(command -v ic-cdk-optimizer)" || "$(ic-cdk-optimizer --version)" != "ic-cdk-optimizer 0.3.4" ]]
```

Reply:
Good catch! Your patch is what `scripts/build` looks like.